curriculum/challenges/english/07-scientific-computing-with-python/learn-interfaces-by-building-an-equation-solver/66793c5b4bdacc17c40ff8e7.md

```diff
@@ -11,7 +11,15 @@ Another feature of f-strings enables you to convert the content of the replaceme
 
 From now on, you'll keep building the output by concatenating strings to `output_string`.
 
-Create a string containing the string representation of your equation centered in a width of `24` characters. Make the string begin and end with two newline characters, and add your new string to the current value of `output_string`.
+Create a string containing the string representation of your equation centered in a width of `24` characters. Make the string begin and end with two newline characters, and add your new string to the current value of `output_string`. The output of `solver` function should look like this:
+
+```py
+
+----Linear Equation-----
+
+       2x +1 = 0        
+
+```
```

Comment:
````suggestion
Create a string containing the string representation of your equation centered in a width of `24` characters. Make the string begin and end with two newline characters, and add your new string to the current value of `output_string`.

The output of `solver` function should look like this:

```md

----Linear Equation-----

       2x +3 = 0        

```
````
Changing the language to `md` to avoid sintax highlighting.